app/Http/Middleware/EnsureEmailIsVerified.php

```diff
@@ -4,22 +4,24 @@
 
 use App\Exceptions\TranslatableException;
 use Closure;
+use Illuminate\Http\Request;
+use Symfony\Component\HttpFoundation\Response;
 
-class EmailVerified
+class EnsureEmailIsVerified
```

Comment:
I'm suggesting overriding the original built-in `EnsureEmailIsVerified` (as we already do) but keeping its name -- as Laravel Ecosys usually does when they override a `Base*` class.